src/applications/mhv/medications/containers/RefillPrescriptions.jsx

```diff
@@ -150,6 +149,25 @@ const RefillPrescriptions = ({ refillList = [], isLoadingList = true }) => {
     [isLoading],
   );
 
+  useEffect(
+    () => {
+      if (refillablePrescriptionsList) {
+        const fullList = refillablePrescriptionsList.sort((a, b) =>
+          a.prescriptionName.localeCompare(b.prescriptionName),
+        );
+        const [refillableList, renewableList] = fullList.reduce(
+          categorizePrescriptions,
+          [[], []],
+        );
+        setFullRefillList(refillableList);
+        setFullRenewList(renewableList);
+      }
+      // disabled warning: fullRefillList must be left of out dependency array to avoid infinite loop
+      // eslint-disable-next-line react-hooks/exhaustive-deps
+    },
+    [refillablePrescriptionsList, prescriptionsApiError],
```

Comment:
if this fails and we already have the list in state, it shows a half error state:
![Screenshot 2024-05-07 at 5 24 00 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/146013972/0e18c830-4d75-4099-9648-2fd70bc2571f)

When I refresh the page:
![Screenshot 2024-05-07 at 5 24 41 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/146013972/36bca0b2-2333-41f7-9bc3-f47b4e46bb69)